ca_tools/scripts/draw_square.py

```diff
@@ -0,0 +1,214 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+
+
+import curses
+import math
+
+import rospy
+from geometry_msgs.msg import Twist, Pose, Quaternion, PoseWithCovariance
+from nav_msgs.msg import Odometry
+from tf.transformations import euler_from_quaternion as efq
+from tf.transformations import quaternion_from_euler as qfe
+
+ANGLE_THRESHOLD = 0.05
```

Comment:
Don't create global constant variables. Move them inside the class IF they only are being used there.